src/rdpc/query/types.ts

```diff
@@ -0,0 +1,24 @@
+export type AnalysisFilterQueryVar = {
+  analysisType?: string;
+  analysisState?: string;
+  studyId?: string;
+  donorId?: string;
+};
+
+export type PageQueryVar = {
+  from: number;
+  size: number;
+};
+
+export type QueryVariable = {
+  analysisFilter: AnalysisFilterQueryVar;
+  analysisPage: PageQueryVar;
+  workflowRepoUrl?: string;
+};
+
+export const retryConfig = {
```

Comment:
This isn't a type, its configuration values. It is good that you moved it into a common location for many places to reference, but it doesn't belong in this file.